FORMULA

```diff
@@ -0,0 +1,9 @@
+name: fluentd
+os: RedHat, CentOS, Ubuntu, Debian
+os_family: RedHat, Debian
+version: 201602
+release: 1
+summary: SaltStack formula for installing and configuring FluentD
+description: SaltStack formula for installing and configuring FluentD
+top_level_dir: fluentd
+recommended: 
```

Comment:
Yes. This is how you enable a formula to be buildable by the SPM tool. https://docs.saltstack.com/en/latest/topics/spm/index.html